workflow-templates/assets/cobra/docsgen/go.mod

```diff
@@ -2,7 +2,7 @@
 // TODO: replace MODULE_NAME with the project's module name
 module MODULE_NAME/docsgen
 
-go 1.14
+go 1.16
```

Comment:
I realize this change does not necessarily follow from the others, but it seems that in this case there are only downsides to indicating compatibility with a Go version we don't do any testing for.